content/posts/looker-pdt-to-dbt-part-one.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Converting Looker PDTs to dbt models: Reasons why (and why not)"
+date: "2018-12-25"
+author: "Dylan Baker"
+---
+
+Over the past few years, Looker has become a market-leading BI tool. Its data governance and self-service capabilities have now become paramount to thousands of BI stacks. For many, the Third Wave of BI (TM Frank Bien) is well and truly here.
+
+Persistent derived tables (PDTs) are a big part of Looker's success. They allow businesses to quickly transform their data with simple SQL `select` statement, abstracting lots of the data engineering that might otherwise be required. They allow you to easily materialize those models as tables in your data warehouse, speeding up the query times for your end users. 
+
+They are addictive. Build a few PDTs, define an explore. Push. Build a few more. Push. Re-define the calculation behind a dimension. Push. Like most addictions though, there comes a point where you realise it may be detrimental to your health and well-being. Well, that of your BI stack anyway. 
```

Comment:
For the first "Push." I'd write "Push to production" for clarity. For the subsequent ones, I think "Push." has better rhythm.
```suggestion
 They are addictive. Build a few PDTs, define an explore. Push to production. Build a few more. Push. Re-define the calculation behind a dimension. Push. Like most addictions though, there comes a point where you realise it may be detrimental to your health and well-being. Well, that of your BI stack anyway. 
```